ui/cypress/integration/volume/createvolume.spec.js

```diff
@@ -0,0 +1,70 @@
+beforeEach(() => {
+  cy.setupMocks();
+  cy.login();
+  cy.visit('/volumes/createVolume');
+});
+
+const VOLUME_NAME = 'test-volume-sparse';
+const STORAGECLASS = 'metalk8s';
+const VOLUME_LABEL_NAME = 'kubernetes.io/name';
+const VOLUME_LABEL_INVALID = 'kubernetes.io&/name';
+const VOLUME_LABEL_VALUE = 'test';
+const VOLUME_TYPE = 'sparseLoopDevice';
+const VOLUME_SIZE = '1 GiB';
+
+// interaction test for creating volume
+describe('the create button in create volume page', () => {
+  it('should be enabled and the Add button should be disabled when no labels is specified', () => {
+    cy.get('input[name=name]').type(VOLUME_NAME);
+
+    cy.get('.sc-select').eq(0).click();
+    cy.get('.sc-select__menu').eq(0).click();
+    cy.get('.sc-select').eq(1).click();
+    cy.get('.sc-select__menu')
```

Comment:
We should avoid dom selectors by classnames and tests this as user interact with. With the current implementation the most elegant solution is to look for elements with text matching /node*/i and click on the next `.sc-select`. Ideally we should add an aria role with listbox value on the dropdown component https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role + an aria-label with value "node"